packages/lib-runtime/src/PluginStore.ts

```diff
@@ -77,11 +83,13 @@ export class PluginStore<TLoadedExtension extends LoadedExtension> {
    */
   addPlugin(metadata: PluginMetadata, extensions: TLoadedExtension[]) {
     if (this.loadedPlugins.has(metadata.name)) {
+      // eslint-disable-next-line no-console
```

Comment:
The `no-console` rule can be disabled once, for the whole file.

Reply:
Do not recommend whole file disabling of lints unless it is a legacy file that predates configurations or there is a unique use-case for it (nothing comes to the top of my head). However, uniquely, `no-console` should always be on for a file. It forces the calls of `console.` to be validated by the developer. Debug statements always will be things we want to vet so we don't leak those into the build.

Reply:
Fair enough, I'll move back to disable-per-line for `no-console` rule.